### DISCUSSION TITLE
커머스솔루션 심사 후 승인 옵션 사용 관련 문의

### DISCUSSION THREAD

**Discussion by gusals0422** [commerce-api-naver/commerce-solutions#34] [묻고 답하기]
안녕하세요

커머스솔루션 개발과정 중 심사 후 승인 옵션 사용관련해서 질문드립니다.

사용자가 솔루션 사이트에서 판매자 확인을 위한 커머스ID 인증 진행시

커머스 ID 인증팝업이 표시된 후 이후 프로세스를 진행하는것으로 이해하였는데

커머스 ID 인증팝업과 관련한 내용은 어디서 확인을 할 수 있을까요?

추가적으로 인증팝업에서 올바른 인증이 이루어졌을 시에

내 솔루션에 등록되어진 Redirect URL 로 판매자 정보가 담긴 JWE 값이 리턴되는 방식이 맞는지도 알고싶습니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

### 판매자 커머스ID 인증 내용 확인
'판매자 커머스ID 인증'과 관련한 기본 내용은 커머스API센터에 게시된 [개발 가이드](https://apicenter.commerce.naver.com/ko/basic/solution-doc#section/%EA%B0%9C%EB%B0%9C%ED%95%98%EA%B8%B0/%ED%8C%90%EB%A7%A4%EC%9E%90-%EC%BB%A4%EB%A8%B8%EC%8A%A4-%EC%95%84%EC%9D%B4%EB%94%94-%EC%9D%B8%EC%A6%9D) 내용을 확인해주시기 바랍니다.
해당 문서의 내용을 숙지하셨다는 가정하에 샘플 목업은 [Github Wiki 문서](https://github.com/commerce-api-naver/commerce-solutions/wiki/%5B%EB%A6%AC%EC%86%8C%EC%8A%A4%5D-%ED%8C%90%EB%A7%A4%EC%9E%90-%EC%BB%A4%EB%A8%B8%EC%8A%A4ID-%EC%9D%B8%EC%A6%9D-%EC%83%98%ED%94%8C) 내용으로 확인이 가능합니다.

> [!NOTE]
> 위 답변에 링크된 Github Wiki 문서는 차후 커머스API센터에서 이관되어 제공될 수 있습니다.
> 답변 게시 후 상당 기간이 지난 시점에 이 답변을 확인하실 경우 커머스API센터 내 개발 가이드를 우선하여 참고해주시기 바랍니다.

### JWE 값 확인 방법
사용자의 '판매자 커머스ID 인증'이 성공적으로 진행되었다면 사용자 웹 브라우저에서 솔루션에 등록된 Redirect URL로 랜딩합니다.
이때 URL 파라미터 형식으로 인증한 판매자의 사용자 정보가 포함된 JWE 토큰 값이 전달됩니다.

해당 토큰은 암호화되어있으며 토큰에 포함된 사용자 정보는 아래 API를 호출하여 확인합니다.
- [커머스솔루션] 판매자 인증 JWE 해석 API (`GET /v1/commerce-solutions/seller-info-by-token`)
